content/tutorials/esc-version-tagging/index.md

```diff
@@ -0,0 +1,254 @@
+"Good day, world!"
+```
+
+Next, initialize new Node.js project and install the [Pulumi ESC SDK for Node.js](/docs/esc/development/languages-sdks/javascript/):
+
+```bash
+$ npm init -y
+$ npm install @pulumi/esc-sdk
+$ touch index.js
+```
+
+Copy the following script into `index.js` and save the file, substituting your Pulumi organization for `tutorials`. The script starts five Node.js web servers (to simulate multiple running applications) that handle requests by fetching and returning the `active` greeting:
+
+```javascript
+const http = require("http");
+const esc = require("@pulumi/esc-sdk");
+
+const startServers = () => {
+
+    // Configure and instantiate the ESC client.
+    const config = new esc.Configuration({ accessToken: process.env.PULUMI_ACCESS_TOKEN })
+    const client = new esc.EscApi(config);
+
+    // Start five web servers on successive ports, beginning with 8080.
+    for (let i = 0; i < 5; i++) {
+        const port = 8080 + i;
+
+        http.createServer(async (req, res) => {
+
+            // Fetch and return the `active` message of the day.
+            const result = await client.openAndReadEnvironmentAtVersion(
+                "tutorials", "messages", "dev", "active"
+            );
+
+            res.end(result.values.motd);
+        }).listen(port);
+
+        console.log(`Listening on http://localhost:${port}`);
+    }
+}
+
+startServers();
+```
+
+Making sure you're in the same shell into which you exported your `PULUMI_ACCESS_TOKEN` earlier, run the script with Node.js, and see that five HTTP servers are now listening on five different local ports:
+
+```bash
+$ node index.js
+
+Listening on http://localhost:8080
+Listening on http://localhost:8081
+Listening on http://localhost:8082
+Listening on http://localhost:8083
+Listening on http://localhost:8084
+```
+
+In a new terminal session, run the following Bash `for` loop, which `curl`s each endpoint and shows that the active MOTD is returned:
+
+```bash
+$ for i in {0..4}; do curl "http://localhost:$((8080 + i))"; done
+
+Good day, world!
+Good day, world!
+Good day, world!
+Good day, world!
+Good day, world!
+```
+
+Finally, update the `active` tag (either with the CLI as below or in the Pulumi Cloud console) to point to the latest version of the environment, then run the loop again to see that all applications have now been reconfigured:
+
+```bash
+$ esc env version tag tutorials/messages/dev@active @4
+
+$ for i in {0..4}; do curl "http://localhost:$((8080 + i))"; done
+
+Lovely to see you, world!
+Lovely to see you, world!
+Lovely to see you, world!
+Lovely to see you, world!
+Lovely to see you, world!
+```
+
+You can stop the server with `Control-C`.
+
+## Wrapping up
+
+In this tutorial, you learned how to use Pulumi ESC's version-tagging features and the ESC SDK for Node.js to apply real-time configuration changes to multiple running applications simultaneously --- without having to touch the source code of any of those applications or redeploy them.
```

Comment:
replace em dash w/ a comma

Reply:
I'll leave this one in, just for the extra bit of emphasis.